### DISCUSSION TITLE
Disable Exceptions with Suffix

### DISCUSSION THREAD

**Discussion by Realium3D** [Mustard2/MustardUI#101] [Ideas]
I've been working on ways to improve the diffeomorphic and MHX viewport performance by converting most of my customization morphs into pure shapekeys to help reduce the number of drivers.

One downside of the enable/disable morphs is that any morphs or shapekeys that change the body shape also become disabled.

I have noticed that when importing custom morphs or shapekeys via diffeomorphic we can assign a suffix to them. This could be really useful for adding those morphs to the "Disable Exceptions" list in MustardUI however it only currently supports prefix which diffeomorphic doesn't have an option for.

I'm not sure how feasible it would be to add suffix support to the disable exceptions list, my python knowledge is sadly limited.

**Comment by Mustard2** [comment]
Hi! The disable exceptions works even if you don't put the full name (it checks if the string you write is in the morph name).
Do you have an example model I can check?

**Comment by Mustard2** [comment]
Closing this as it's very old.

Please let me know opening a new Issue if there is still the necessity of some changes
